main.css

```diff
@@ -0,0 +1,550 @@
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 80vh;
+
+}
+
+.form-container {
+    display: flex;
+    place-content: center;
+}
+
+.form-container__wrapper {
+    width: 80%;
+}
+
+.form-group {
+    display: flex;
+    gap: 10px;
+    padding: 0.5rem 0rem;
+}
+
+.form-input__control {
+    padding: 0.5rem 1.5rem;
+}
+
+.form-container__text {
+    padding: 1rem 0rem;
+}
+
+.form-container__sub-heading {
+    font-weight: 700;
+    font-size: 13px;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    color: #252B42;
+
+}
+
+.form-container__heading {
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 57px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+}
+
+.form-input__select {
+    width: 13.4rem;
+    height: 2rem;
+}
+
+.col-75 {
+    width: auto;
+    margin-top: 6px;
+
+}
+
+.client-card {
+    display: flex;
+    flex-direction: column;
+    width: 20rem;
+    background-color: white;
+    border: 1px solid #DEDEDE;
+    border-radius: 1rem;
+    padding: 1rem;
+}
+
+
+.client-card__ratings {
+    display: flex;
+}
+
+.client-card__profile {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+
+.client-card__picture {
+    height: 4rem;
+}
+
+.client-card__names {
+    padding: 0;
+    margin: 0;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+    color: #FFA62B;
+
+}
+
+.client-card__para {
+    padding: 20px 0px;
+}
+
+.client-card__profession {
+    padding: 0;
+    margin: 0;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+}
+
+.form-text {
+    width: 43.5%;
+}
+
+.btn--colored {
+    width: 20%;
+    margin-top: 1.5rem;
+    border-radius: 0;
+}
+
+.sub-footer {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 2rem 0;
+}
+
+.btn--sm {
+    padding: 10px 20px;
+    width: 10%;
+    margin-top: 0rem;
+}
+
+.title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    letter-spacing: 0.1px;
+    color: #252B42;
+}
+
+.sub-title {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+
+.footer {
+    height: 15vh;
+    background-color: #252B42;
+    display: flex;
+    place-content: center;
+    place-items: center;
+}
+
+.footer__content {
+    width: 80%;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #364067;
+    padding: 1.5rem 0rem;
+}
+
+.footer__content--unbordered {
+    border-bottom: none;
+
+}
+
+.footer--start-placed-content {
+    height: 35vh;
+    padding: 0rem 0rem;
+    background-color: #252B42;
+    display: flex;
+    place-content: center;
+    place-items: start;
+}
+
+.footer--colored {
+    height: 10vh;
+    overflow-y: hidden;
+    padding: 0rem 0rem;
+    background-color: #16697A;
+}
+
+.logo--sm {
```

Comment:
To structure your css it is better to put your block, block element, and modifier in the same places for instance you can have this `logo--sm` in the same place as where you defined your block